youtube_dl/extractor/lemonde.py

```diff
@@ -1,58 +1,64 @@
         },
     }, {
-        # standard iframe embed
+        # Dailymotion embeded
         'url': 'http://www.lemonde.fr/les-decodeurs/article/2016/10/18/tout-comprendre-du-ceta-le-petit-cousin-du-traite-transatlantique_5015920_4355770.html',
         'info_dict': {
-            'id': 'uzsxms',
+            'id': 'x4yejm2',
             'ext': 'mp4',
-            'title': "CETA : quelles suites pour l'accord commercial entre l'Europe et le Canada ?",
-            'thumbnail': r're:^https?://.*\.jpg',
-            'duration': 325,
+            'title': "CETA : quelles suites l'accord commercial entre l'Europe et le Canada ?",
+            'duration': 326,
             'upload_date': '20161021',
-            'timestamp': 1477044540,
-            'uploader_id': '3pmkp',
-        },
-        'params': {
-            'skip_download': True,
-        },
-    }, {
-        'url': 'http://redaction.actu.lemonde.fr/societe/video/2016/01/18/calais-debut-des-travaux-de-defrichement-dans-la-jungle_4849233_3224.html',
-        'only_matching': True,
-    }, {
-        # YouTube embeds
-        'url': 'http://www.lemonde.fr/pixels/article/2016/12/09/pourquoi-pewdiepie-superstar-de-youtube-a-menace-de-fermer-sa-chaine_5046649_4408996.html',
-        'only_matching': True,
+            'timestamp': 1477044897,
+            'uploader_id': 'x1wd0c',
+            'uploader': 'Le Monde.fr',
+            'description': 'CETA. 4 lettres pour Comprehensive Economic and Trade Agreement, ou Accord économique et commercial global). Un nom un peu barbare pour un traité transatlantique, cousin du Tafta, qui agite en ce moment l’actualité européenne.\xa0Pour comprendre les enjeux de cet accord commercial, notre spécialiste Maxime Vaudano.'
+        }
     }]
 
+    _PROVIDERS = {
+        "youtube": "https://www.youtube.com/embed/{}",
+        "dailymotion": "https://www.dailymotion.com/video/{}"
+    }
+
     def _real_extract(self, url):
         display_id = self._match_id(url)
-
         webpage = self._download_webpage(url, display_id)
 
-        digiteka_url = self._proto_relative_url(self._search_regex(
```

Comment:
Do not touch this.

Reply:
This is not up to date with the website, it always return `None`.